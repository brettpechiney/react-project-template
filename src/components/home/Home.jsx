import React from 'react';
import styled from 'styled-components';

const Home = () => (
  <Wrapper>
    <p className="intro">
      This application was made with{' '}
      <span role="img" aria-label="love">
        ❤️
      </span>
      by Brett
    </p>
  </Wrapper>
);

const Wrapper = styled.section`
  display: flex;
  align-items: center;
  justify-content: center;

  .intro {
    font-size: large;
  }
`;

export default Home;
