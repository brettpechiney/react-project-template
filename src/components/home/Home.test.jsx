import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);

  const wrapper = shallow(<Home {...props} />);

  return {
    props,
    wrapper,
  };
};

describe('<Home />', () => {
  const { wrapper } = setup();

  it('renders', () => {
    expect(wrapper.exists()).toBe(true);
  });
});
