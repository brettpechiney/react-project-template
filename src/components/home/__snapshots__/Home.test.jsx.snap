// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correctly 1`] = `
<section
  className="Home__Wrapper-sc-2dee5m-0 cjgKvf"
>
  <p
    className="intro"
  >
    This application was made with
     
    <span
      aria-label="love"
      role="img"
    >
      ❤️
    </span>
    by Brett
  </p>
</section>
`;
